R/check-cran.sh

```diff
@@ -65,6 +65,10 @@ fi
 
 echo "Running CRAN check with $CRAN_CHECK_OPTIONS options"
 
+# Remove this environment variable to allow check suggested packages once
+# Jenkins installs arrow. See SPARK-29339
+_R_CHECK_FORCE_SUGGESTS_=false
```

Comment:
I manually checked that CRAN check passes in my local.

In Jenkins, it seems failed because Arrow R library itself is not installed in Jenkins:

```
Package suggested but not available: ���arrow���

The suggested packages are required for a complete check.
Checking can be attempted without them by setting the environment
variable _R_CHECK_FORCE_SUGGESTS_ to a false value.
```

Reply:
I plan to change the minimum version soon (as it's experimental, it might be fine). I will work with Shane to install Arrow R library in Jenkins after that.

Reply:
@BryanCutler, BTW, shall we upgrade Arrow in Spark from 0.12 to 0.14 at `pom.xml`?